invisible_cities/cities/beersheba.py

```diff
@@ -409,9 +416,9 @@ def beersheba(files_in, file_out, compression, event_range, print_mod, detector_
 
     with tb.open_file(file_out, "w", filters = tbl.filters(compression)) as h5out:
         # Define writers
-        write_event_info = fl.sink(run_and_event_writer(h5out), args=("run_number", "event_number", "timestamp"))
-        write_deconv     = fl.sink(  deconv_writer(h5out=h5out), args =  "deconv_dst"         )
-        write_summary    = fl.sink( summary_writer(h5out=h5out), args =  "summary"            )
+        write_event_info = fl.sink(run_and_event_writer (h5out), args=("run_number", "event_number", "timestamp"))
```

Comment:
since you already touched the white spaces, maybe you can align `=` in which case you should put white space before and after `=`